lib/gui/test_swatches.py

```diff
@@ -0,0 +1,189 @@
+        options_sizer.Add(grid_main_sizer, 0, wx.ALL, 10)
+
+        param_label = wx.StaticText(self.options, wx.ID_ANY, _("Param"))
+        grid_main_sizer.Add(param_label, 0, 0, 0)
+
+        self.param = wx.ComboBox(self.options, wx.ID_ANY, choices=[])
+        for choice in self.choices:
+            self.param.Append(choice.name, choice)
+        self.param.SetSelection(0)
+        grid_main_sizer.Add(self.param, 0, 0, 0)
+
+        start_value_label = wx.StaticText(self.options, wx.ID_ANY, _("Start Value"))
+        grid_main_sizer.Add(start_value_label, 0, 0, 0)
+
+        self.start_value = wx.SpinCtrlDouble(self.options, wx.ID_ANY, initial=2.5, min=0.0, max=500.0)
+        self.start_value.SetDigits(2)
+        grid_main_sizer.Add(self.start_value, 0, 0, 0)
+
+        step_label = wx.StaticText(self.options, wx.ID_ANY, _("Increase by"))
+        grid_main_sizer.Add(step_label, 0, 0, 0)
+
+        self.step = wx.SpinCtrlDouble(self.options, wx.ID_ANY, initial=0.5, min=0.01, max=500.0)
+        self.step.SetDigits(2)
+        grid_main_sizer.Add(self.step, 0, 0, 0)
+
+        cols_label = wx.StaticText(self.options, wx.ID_ANY, _("Columns"))
+        grid_main_sizer.Add(cols_label, 0, 0, 0)
+
+        self.columns = wx.SpinCtrl(self.options, wx.ID_ANY, "0", min=1, max=100)
+        grid_main_sizer.Add(self.columns, 0, 0, 0)
+
+        rows_label = wx.StaticText(self.options, wx.ID_ANY, _("Rows"))
+        grid_main_sizer.Add(rows_label, 0, 0, 0)
+
+        self.rows = wx.SpinCtrl(self.options, wx.ID_ANY, "0", min=1, max=100)
+        grid_main_sizer.Add(self.rows, 0, 0, 0)
+
+        spacing_label = wx.StaticText(self.options, wx.ID_ANY, _("Spacing"))
+        grid_main_sizer.Add(spacing_label, 0, 0, 0)
+
+        self.spacing = wx.SpinCtrlDouble(self.options, wx.ID_ANY, initial=5, min=0.0, max=100.0)
+        self.spacing.SetDigits(2)
+        grid_main_sizer.Add(self.spacing, 0, 0, 0)
+
+        self.info = wx.Panel(self.notebook, wx.ID_ANY)
+        self.notebook.AddPage(self.info, _("Help"))
+
+        info_sizer = wx.BoxSizer(wx.VERTICAL)
+
+        info_text_label = wx.StaticText(
+            self.info,
+            wx.ID_ANY,
+            _("This extension generates test swatches from a selection.\n"
+              "Test swatches help to find best stitch settings for your design.\n"
+              "Sew them out with the same thread and fabric as the final designs."),
+            style=wx.ALIGN_LEFT
+        )
+        info_text_label.Wrap(500)
+        info_sizer.Add(info_text_label, 0, wx.ALL, 8)
+
+        info_sizer.Add((20, 20), 0, 0, 0)
+
+        more_info_label = wx.StaticText(self.info, wx.ID_ANY, _("Get more information on our website"))
+        info_sizer.Add(more_info_label, 0, wx.ALL, 8)
+
+        self.help_link = wx.adv.HyperlinkCtrl(
+            self.info,
+            wx.ID_ANY,
+            "https://inkstitch.org/docs/edit/#generate-test-swatches-from-selection",
```

Comment:
Is it okay that this links to the English version for everyone?

Reply:
Ah no :)